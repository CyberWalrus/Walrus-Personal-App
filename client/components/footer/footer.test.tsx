import * as Enzyme from "enzyme";
import { shallow } from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";
import toJson, { Json } from "enzyme-to-json";
import * as React from "react";
import Footer from "./footer";

Enzyme.configure({adapter: new Adapter()});
it(`Footer correctly render`, (): void => {
  const tree: Json = toJson(shallow(<Footer />));
  expect(tree).toMatchSnapshot();
});
