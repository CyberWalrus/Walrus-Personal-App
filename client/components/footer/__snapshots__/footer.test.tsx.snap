// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer correctly render 1`] = `
<footer
  className="page-footer"
>
  <div
    className="logo logo_footer"
  >
    <Link
      className="logo__link"
      to="/"
    >
      <img
        className="logo__img"
        src="img/walrus-icon-white.png"
      />
    </Link>
  </div>
  <div
    className="copyright"
  >
    <p>
      Â© 2019 Walrus Personal App
    </p>
  </div>
</footer>
`;
